compiler/next/include/chpl/queries/stringify-functions.h

```diff
@@ -359,6 +359,14 @@ template<typename... ArgTs> struct stringify<std::tuple<ArgTs...>> {
                           std::index_sequence_for<ArgTs...>{});
   }
 };
+
+
+// force this to always build so it is available during debug
```

Comment:
maybe put a TODO comment here, or maybe leave this out entirely. We don't know yet if the approach works in a debugger.